src/library/autodj/autodjprocessor.cpp

```diff
@@ -697,6 +692,10 @@ void AutoDJProcessor::playerPositionChanged(DeckAttributes* pAttributes,
             } else {
                 // At least right deck is playing
                 // Set crossfade thresholds for right deck.
+                if (sDebug) {
+                    qDebug() << this << "playerPositionChanged"
```

Comment:
consider making these `if constexpr`. This conveys intent better (the fact that whether this "branch" is taken is supposed to be made at compile time), and decreases the work for the optimizer. 
```suggestion
                if constexpr (sDebug) {
                    qDebug() << this << "playerPositionChanged"
```